Sources/Styleguide/PointFreeBaseStyles.swift

```diff
@@ -9,6 +9,7 @@ public enum Colors {
   public static let gray150 = Color.other("#242424")
   public static let gray300 = Color.other("#555555")
   public static let gray400 = Color.other("#666666")
+  public static let gray500 = Color.other("#808080")
```

Comment:
I added this middle of the road gray to our style guide because we've needed something similar a number of times but have resorted to inline styles. this causes a bit of churn in the snapshots.